minimax.py

```diff
@@ -0,0 +1,26 @@
+import math 
+
+''' Minimax helps to achieve maximum score in a game by checking all possible moves
+    depth is current depth in game tree. 
+    nodeIndex is index of current node in scores[].
+    if move is of maximizer return true else false
+    leaves of game tree is stored in scores[]  
+    height is maximum height of Game tree
+'''
+
+def minimax (Depth, nodeIndex, isMax, scores, height):  
+
+    if (Depth == height):  
+        return scores[nodeIndex] 
+
+    if (isMax): 
+        return max(minimax(Depth + 1, nodeIndex * 2, False, scores, height), minimax(Depth + 1, nodeIndex * 2 + 1, False, scores, height)) 
```

Comment:
These lines are too long to be read on GitHub without scrolling left and right so wrap the whole return value in () and then wrap the lines after the comma to improve readability.